debian/kolibri.scripts-common

```diff
@@ -346,15 +346,21 @@ kolibri_debconf_set_defaults()
         db_fset kolibri/pre-010-upgrade-system-user seen true
     fi
 
-    if [ "$USER_SEEN" == "false" ]
+    if [ -s /etc/kolibri/username ]    # Test that file exists with non-zero size
```

Comment:
You can simplify this change quite a lot by adding the test to the list of tests in [line 341](https://github.com/learningequality/kolibri-installer-debian/pull/117/files#diff-c957f55a152335a0eab960eca7417081d595632ad57f1efca7a51c9fa4a83969R341) AFAICT

Reply:
I saw that earlier, yes.

But I chose to keep the change more localized & more legible.

i.e. to make clear this PR is basically a 2-line change wrapped in an if statement.

(@jredrejo is of course free to move things around as he sees fit!)